zlp-scheduler/db/seeds.rb

```diff
@@ -6,6 +6,29 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
+def create_schedule(student,schedule_name, classes)
+        @schedule = Schedule.new
+        @schedule.update_attributes(:name => schedule_name)
+        student.schedules.push(@schedule)
+        classes.each do |class_hash|
+                @course = Course.where(class_hash)
+                @schedule.courses.push(@course)
+        end
+        
+        @schedule.save
+        @user.schedules.push(@schedule)   
+end
+
+
+@terms = Term.ImportTermList!
```

Comment:
We really shouldn't be hitting the api (which this function does) in testing. And this seed data will probably be loaded every time a test is run (assuming I'm remembering how the testing stuff works correctly). So, we should probably just hard-code some individual terms and set one of them to be active.